tone-analyzer/src/test/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/ToneAnalyzerTest.java

```diff
@@ -18,6 +18,7 @@
 
 import java.io.IOException;
 
+import com.ibm.watson.developer_cloud.tone_analyzer.v3.model.UtterancesTone;
```

Comment:
Is there a way to a shorter import? All the other Java classes from the model package are also being imported this way.